src/app/login-page/login-page.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss'],
+})
+export class LoginPageComponent {
+  loginForm: FormGroup;
+  constructor(private fb: FormBuilder, private router: Router) {
+    localStorage.clear();
+    this.loginForm = this.fb.group({
+      login: ['', Validators.required],
+      password: ['', Validators.required],
+    });
+  }
+
+  get login() {
+    return this.loginForm.get('login');
+  }
+
+  get password() {
+    return this.loginForm.get('password');
+  }
+
+  onLoanFormSubmit() {
+    if (
+      this.login?.value == 'admin' &&
+      this.password?.value == 'admin' &&
+      this.loginForm.valid
+    ) {
+      localStorage.setItem(
+        'token',
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
```

Comment:
That's an interesting JWT. Is this an example taken from the backend?

Reply:
I just went to a site for generating JWT tokens and took this example token from there just to replicate real life tokens